content/_data/changelogs/lts.yml

```diff
@@ -4785,6 +4785,377 @@
+    - jeffret-b
+    references:
+      - pull: 5043
+      - url: "https://github.com/jenkinsci/remoting/releases/tag/remoting-4.6"
+        title: Remoting 4.6 changelog
+    message: |-
+      Upgrade from Remoting 4.5 to Remoting 4.6 with bugfixes and dependency updates.
+
+  - type: rfe
+    category: rfe
+    pull: 5111
+    authors:
+    - timja
+    references:
+      - pull: 5111
+      - url: "https://github.com/stapler/stapler/releases/tag/stapler-parent-1.262"
+        title: Stapler 1.262 changelog
+    message: |-
+      Update stapler to 1.262 to fix a number of IllegalReflectiveAccessWarnings when running on Java 11.
+
+  - type: rfe
+    category: rfe
+    pull: 5129
+    authors:
+    - dependabot[bot]
+    references:
+      - pull: 5129
+      - url: "https://github.com/jnr/jnr-posix/compare/jnr-posix-3.0.45...jnr-posix-3.1.4"
+        title: Commits from jnr-posix 3.0.45 to 3.1.4
+    message: |-
+      Update jnr-posix library from 3.0.45 to 3.1.4.
+
+  - type: rfe
+    category: rfe
+    pull: 5125
+    authors:
+    - dependabot[bot]
+    references:
+      - pull: 5125
+      - url: "https://github.com/java-native-access/jna/blob/master/CHANGES.md#release-560"
+        title: JNA 5.6.0 changelog
+      - url: "https://github.com/java-native-access/jna/blob/master/CHANGES.md#release-550"
+        title: JNA 5.5.0 changelog
+      - url: "https://github.com/java-native-access/jna/blob/master/CHANGES.md#release-540"
+        title: JNA 5.4.0 changelog
+    message: |-
+      Update Java native access (jna) library from 5.3.1 to 5.6.0 for more recent platform library fixes and enhancements.
+
+  - type: major rfe
+    category: developer
+    pull: 4944
+    issue: 19561
+    authors:
+    - jglick
+    references:
+      - pull: 4944
+      - pull: 5115
+      - url: "https://github.com/jenkinsci/jep/blob/master/jep/227/README.adoc"
+        title: JEP-227
+      - url: "/blog/2020/11/10/spring-xstream/"
+        title: Spring and XStream updates (breaking changes!) blog post
+      - url: "https://x-stream.github.io/changes.html#1.4.14"
+        title: XStream 1.4.14 changelog
+      - url: "https://x-stream.github.io/changes.html#1.4.15"
+        title: XStream 1.4.15 changelog
+    message: |-
+      Developer: Jenkins now uses an updated version of the XStream serialization library without custom patches.
+
+  - type: major rfe
+    category: developer
+    pull: 4848
+    authors:
+    - jglick
+    references:
+      - pull: 4948
+      - pull: 5166
+      - url: "https://github.com/jenkinsci/jep/blob/master/jep/228/README.adoc"
+        title: JEP-228
+      - url: "/blog/2020/11/10/spring-xstream/"
+        title: Spring and XStream updates (breaking changes!) blog post
+      - url: https://github.com/spring-projects/spring-security/releases/tag/5.4.2
+        title: Spring Security 5.4.2 release notes
+    message: |-
+      Developer: Jenkins now uses Spring Security rather than its predecessor, Acegi Security.
```

Comment:
Wrong sentence form

Reply:
Corrected in 8b7a11c223d8bb0f22316a2a88035ffdeb903f1c